pkg/instance/repository.go

```diff
@@ -132,16 +183,16 @@ func (r repository) Unlink(instance *model.Instance) error {
 }
 
 func (r repository) Save(instance *model.Instance) error {
-	key := r.instanceParameterEncryptionKey
+	//key := r.instanceParameterEncryptionKey
 
 	populateParameterRelations(instance)
-
-	err := encryptParameters(key, instance)
-	if err != nil {
-		return err
-	}
-
-	err = r.db.Session(&gorm.Session{FullSaveAssociations: true}).Save(instance).Error
+	/*
+		err := encryptParameters(key, instance)
+		if err != nil {
+			return err
+		}
+	*/
```

Comment:
Are the commented out lines here intentional?

Reply:
The entire method is to be deleted

Reply:
If it should be merged like this I'd add a `// TODO ...` above those lines.